deployment/helm/synopsys-alert/README.md

```diff
@@ -452,6 +455,46 @@ For the on-premise database deployment a second Persistent Volume must be create
     databaseName: <DATABASE_NAME>
   ```
   - Replace <DATABASE_NAME> with the name of the database created in previous steps
+
+- Configure Postgres admin username and password following the instructions outlined here: [Using External Database](#using-external-database)
```

Comment:
Should there be a similar message on line 394 linking to the internal database instructions?

Reply:
I agree we should like to the internal database instructions.